log-viewer/src/main/java/com/logviewer/data2/LogLevels.java

```diff
@@ -0,0 +1,37 @@
+package com.logviewer.data2;
+
+import com.typesafe.config.Config;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class LogLevels {
+    // See org.apache.log4j.lf5.LogLevel, org.apache.log4j.Level, https://en.wikipedia.org/wiki/Syslog
+    private static final String[] LEVELS = {
+            "OFF", "FATAL", "SEVERE", "EMERGENCY", "ALERT", "CRITICAL", "ERROR",
+            "WARN", "WARNING",
+            "INFO", "CONFIG", "NOTICE", "INFORMATIONAL",
+            "DEBUG", "FINE", "FINER",
+            "FINEST", "TRACE", "ALL",
+    };
+
+    private static String[] CUSTOM_LEVELS = new String[0];
```

Comment:
I don't like using a static field to store a level list. I'll try to invent something else. If I can't create another solution, I'll merge the code that uses the static field.

Reply:
I've created `List<String> customLevels` field in AbstractPatternLogFormat. Custom fields can be specified using
```
logs = [
  {
    path: ${HOME}"/my-app/logs/*.log"
    format: {
      type: LogbackLogFormat
      customLevels: ["XXX", "YYY"]
      pattern: "%date{yyyy-MM-dd_HH:mm:ss.SSS} [%thread] %-5level %logger{35} - %X{pipelineId}%X{contentId}%msg%n"
    }
  }
]
```
or
```
log-viewer.log-levels=["TEST"]
```

It's important to have the level list in `LogFormat` object when log-viewer works in multinode mode. The configuration may be different on different nodes, but `LogFormat` will be same, log-viewer sends it to other nodes when it needs to load data from other node.

Reply:
@sevdokimov Great, I've been thinking about that too.